### ISSUE TITLE
Add 

### ISSUE THREAD

**Issue by li2109** [Knowly-ai/langtorch#147]

**Comment by Leowuqunqun** [comment]
@li2109 Prompt lengths don't seem to be consistent across LLMs

**Comment by li2109** [comment]
> @li2109 Prompt lengths don't seem to be consistent across LLMs

Yes, they are not the same. There will be a parameter you can set to adjust

**Comment by li2109** [comment]
> > Yes, they are not the same. There will be a parameter you can set to adjust
> 
> 
> 
> To look into whether the different LLMs provide APIs for token lengths, or do they use magic values in each LLM model configuration? 

For 1) i don't think there's API specifically for this? Well, the prompt length is fixed given the model.
For 2) Magic number won't work as developer eventually will update it.

I think Neither would work. manual input of prompt length probably is the best option here.
Let's say the prompt length is 100 tokens; some people would prefer leaving 50 tokens for history; some might think 80 is better. Current user input also plays a role here.
